platform/_partials/auth/sso/microsoft.mdx

```diff
@@ -8,6 +8,17 @@ import DisablePasswordAuthPartial from "../disable-password-auth.mdx";
 
 vCluster Platform is able to use Microsoft OAuth2 flow to identify the end user through their Microsoft account.
 
+:::warning Tenant Must Be Specified in Microsoft Connector Config to Sync SSO Groups
+groups claim is only supported when tenant is specified in Microsoft connector config. Furthermore, tenant must also be configured to either <tenant uuid> or <tenant name>. In order for vCluster Platform to be able to list groups on behalf of logged in user, an explicit organization administrator consent is required. To obtain the consent do the following:
```

Comment:
```suggestion
groups claim is only supported when tenant is specified in Microsoft connector config. Furthermore, tenant must also be configured to either `<tenant uuid> or <tenant name>`. In order for vCluster Platform to be able to list groups on behalf of logged in user, an explicit organization administrator consent is required. To obtain the consent do the following:
```

Angle brackets need to be enclosed in backticks due to the `jsx` syntax limitations